components/webdriver_server/lib.rs

```diff
@@ -227,6 +227,8 @@ struct Handler {
     current_action_id: Cell<Option<WebDriverMessageId>>,
 
     resize_timeout: u32,
+
+    sent_action_count: RefCell<u32>,
```

Comment:
```suggestion
    /// Number of pending actions of which WebDriver is waiting for responses.
    num_pending_actions: RefCell<u32>,
```
This is neat! Let's improve the clarity a bit.